lib/train/platforms/platform.rb

```diff
@@ -25,7 +26,18 @@ def direct_families
     def name
       # Override here incase a updated name was set
       # during the detect logic
-      @platform[:name] || @name
+      @clean_name
```

Comment:
It's a bit confusing to me to have an instance_var named `@clean_name` and a method named `clean_name` and then use them in different ways.

Can we just default to using the `clean_name` method, and memoize the cleaned_name in that method so we only do the translation once?